src/include/btree_cmp_inline.h

```diff
@@ -14,12 +14,193 @@
 #endif
 #endif
 
-#if defined(HAVE_ARM_NEON_INTRIN_H)
-#include <arm_neon.h>
-#endif
-/* 16B alignment */
-#define WT_ALIGNED_16(p) (((uintptr_t)(p)&0x0f) == 0)
 #define WT_VECTOR_SIZE 16 /* chunk size */
+#define WT_COMPARE_SHORT_MAXLEN 16
+
+#ifdef HAVE_X86INTRIN_H
+
+#define WT_VECTOR_MASK 0xFFFF
+/*
+ * __lex_compare_gt_16 --
+ *     Lexicographic comparison routine for data greater than 16 bytes. Returns: < 0 if user_item is
+ *     lexicographically < tree_item, = 0 if user_item is lexicographically = tree_item, > 0 if
+ *     user_item is lexicographically > tree_item. We use the names "user" and "tree" so it's clear
```

Comment:
Please use words for everything in this comment, rather than a mixture of algebraic symbols and words. Similarly for all the comments.